src/Helpers/AliasEmitters/AliasData.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * AliasData holds the data needed for one site environment alias.
+ */
+
+namespace Pantheon\Terminus\Helpers\AliasEmitters;
+
+class AliasData
```

Comment:
This should be a Pantheon\Terminus\Model belonging to an Environment. (Using EnvironmentInterface and EnvironmentTrait as the Domain model does.)